app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryInfo.js

```diff
@@ -2,16 +2,18 @@ import React from 'react';
 import CollapsedStoryTitle from './CollapsedStoryTitle';
 import CollapsedStoryLabels from './CollapsedStoryLabels';
 import StoryPropTypes from '../../shapes/story';
+import PropTypes from 'prop-types';
 
-const CollapsedStoryInfo = ({ story }) => (
+const CollapsedStoryInfo = ({ story, onClickLabel }) => (
   <div className="Story__info">
-    <CollapsedStoryLabels story={story} />
+    <CollapsedStoryLabels story={story} onClickLabel={onClickLabel} />
     <CollapsedStoryTitle story={story} />
   </div>
 );
 
 CollapsedStoryInfo.propTypes = {
-  story: StoryPropTypes
+  story: StoryPropTypes,
+  onClickLabel: PropTypes.func.isRequired
```

Comment:
I think `onLabelClick` would be better 🤔 

Usually, I create the event's name like `on[Thing Name][Native Event Name]` 
- `onAcceptClick`
- `onButtonHover`
- `onListScroll`

What do you think?

Reply:
Nice! And thank you for share this trick, I liked! :thinking: